webapp/src/ts/effects/contacts.effects.ts

```diff
@@ -52,6 +56,8 @@ export class ContactsEffects {
         this.store.select(Selectors.getForms),
       ),
       exhaustMap(([{ payload: { id, silent } }, userFacilityId, forms]) => {
+        this.selectContactTrack = 'select_contact:load_everything';
+        const trackPerformance = this.performanceService.track(this.selectContactTrack);
```

Comment:
Move this below the `if(!id)` check - there's no point starting the track and then returning before you stop it.